code/game/gamemodes/shadowling/shadowling_items.dm

```diff
@@ -0,0 +1,69 @@
+/obj/item/clothing/under/shadowling
```

Comment:
i think these are mainly there to prevent them from putting other clothes on. also, this file contains the markings effects